evmbin/src/main.rs

```diff
@@ -76,6 +77,9 @@ State test options:
     --only NAME        Runs only a single test matching the name.
     --chain CHAIN      Run only tests from specific chain.
 
+Stats jsontests VM options:
+    --folder           Run jsontest for a folder instead of a single file.
```

Comment:
Could this not be inferred from the input path? If it's a file => run the file; if it's a folder => run all tests within?